src/js/controller/model.js

```diff
@@ -266,8 +271,9 @@ define([
             if (this[attr] === val) {
                 return;
             }
+            var oldVal = this[attr];
             this[attr] = val;
-            this.trigger('change:' + attr, this, val);
+            this.trigger('change:' + attr, this, val, oldVal);
```

Comment:
The backbone way to handle this would be to store the previous attributes value in the model, which are query-able via "model.previous('state')"

For now, this is fine, but something to think about in the future.